.env.example

```
@@ -57,3 +57,8 @@ VITE_PUSHER_HOST="${PUSHER_HOST}"
 VITE_PUSHER_PORT="${PUSHER_PORT}"
 VITE_PUSHER_SCHEME="${PUSHER_SCHEME}"
 VITE_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
+
+TEST_DB_HOST=localhost
+TEST_DB_DATABASE=laravel11
```

```suggestion
TEST_DB_DATABASE=
```

je te user decide kare su rakhvu